frontend/src/scenes/funnels/Funnel.scss

```diff
@@ -3,13 +3,13 @@
 .funnels-empty-state {
     text-align: center;
     h2.funnels-empty-state__title {
-        font-weight: 600;
+        font-weight: bold;
         font-size: 1.5rem;
         line-height: 1.6rem;
     }
 
     .funnels-empty-state__description {
-        font-weight: 500;
+        font-weight: normal;
```

Comment:
I don't think this is needed. Looks like `font-weight: normal;` is the default here.